src/main/java/org/checkerframework/specimin/TargetMethodFinderVisitor.java

```diff
@@ -195,6 +195,7 @@ public Visitable visit(ConstructorDeclaration method, Void p) {
       insideTargetMethod = true;
       targetMethods.add(method.resolve().getQualifiedSignature());
       unfoundMethods.remove(methodName);
+      usedClass.add(method.resolve().getPackageName() + "." + method.resolve().getClassName());
```

Comment:
The code both here and on line 198 calls `method.resolve()` twice. This call is relatively expensive (it invokes the symbol solver), so we should call it once and store the result in a local variable.